web2py/applications/timereporting/models/tablesDef.py

```diff
@@ -1,2 +1,25 @@
-db.define_table('WorkShift', Field('InitialTime', 'datetime'),Field('EndTime', 'datetime'))
-db.define_table('TimeTracker', Field('WorkShift_id', 'reference WorkShift'),Field('user_id','reference auth_user'),Field('Approved','boolean'))
+db.define_table(
+    'WorkShift',
```

Comment:
For workShift, I think for this table, we need to have start_time and end_time for shift recording time fields additional to these fields maybe ids for the student adding the shift.